component-catalog-app/Examples/Tabs.elm

```diff
@@ -122,13 +123,35 @@ example =
                         let
                             code =
                                 [ moduleName ++ ".view"
-                                , "    { title = " ++ Code.maybeString settings.title
-                                , "    , alignment = " ++ moduleName ++ "." ++ Debug.toString settings.alignment
-                                , "    , customSpacing = " ++ Code.maybeFloat settings.customSpacing
-                                , "    , focusAndSelect = identity"
+                                , "    { focusAndSelect = identity"
```

Comment:
I see this was `identity` before your changes, but would it make sense to change it to `FocusAndSelectTab`, or something like that?

Reply:
this is in the example code, right? Probably not, because then we'd need to define that